tests/console/nftables.pm

```diff
@@ -104,6 +111,7 @@ true');
 
     # check firewall settings and save them to logs
     assert_script_run('echo -e "____ firewall services --zone=public ____\n" >> ' . $logs);
+    assert_script_run("firewall-cmd --list-services --zone=public | grep -e dhcpv6-client");
     assert_script_run("firewall-cmd --list-services --zone=public |& tee -a $logs");
```

Comment:
When the output is already forwarded into a file then it's likely faster to just grep the file instead of calling `firewall-cmd` twice